che-plugin-viewer/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
hello, AFAIK it's Eclipse License 2.0 in our Eclipse projects

Reply:
We can make it Eclipse License 2.0. No issues. I will do that in my next PR